### DISCUSSION TITLE
Calendar multiple selection

### DISCUSSION THREAD

**Discussion by miggio90210** [fomantic/Fomantic-UI#2018] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)]
Hi guys, I would like to know if it's possible to have a multiple selection inside an inline calendar.
I'm trying to use 'onChange' to update an array that contains selected dates and then fill eventDates with that array and finally refresh.
But when I want to remove a date from eventDates, clicking on a date, it doesn't desappear from the visual calendar.

**Comment by lubber-de** [comment]
Multiple Date selection inside one single calendar is currently not implemented.
However, removing eventDates by refreshing the calendar with a new eventdates array, should work. Could you please provide a jsfiddle with the code you currently have?

**Comment by otavioserra** [comment]
@lubber-de I made one simple solution using similarity with multi selection dropdown module. You can use it freely if you want.

[https://jsfiddle.net/otavioserra/Lcqzy1hv/11/](https://jsfiddle.net/otavioserra/Lcqzy1hv/11/)
